wicket-tool/src/java/org/sakaiproject/wicket/markup/html/fckeditor/FCKEditorPanel.java

```diff
@@ -36,87 +36,98 @@
 	private transient Logger logger = Logger.getLogger(FCKEditorPanel.class);
 
 	private boolean stripContainingParagraphTags = false;
-	
+
 	public static final String BASIC = "Basic";
 	public static final String DEFAULT = "Default";
-	
+
 	private static final String ESCAPED_OPENING_P = "&lt;p&gt;";
 	private static final String ESCAPED_CLOSING_P = "&lt;/p&gt;";
-	
+
 	/**
 	 * 
 	 * @param id The wicket:id
 	 * @param model The data model
 	 * @param width The width of the rendered textarea
 	 * @param height The height of the rendered text area
-	 * @param toolbarSet The set of toolbars you want rendering, either basic of
-	 * 		             default
+	 * @param toolbarSet The set of toolbars you want rendering, either basic of default
 	 * @param collectionId The Sakai collection id for the FCKeditor to use
-	 * @param strip Set to true if you want the component to strip off the
-	 *              surrounding paragraph tags that FCKeditor produces
+	 * @param strip Set to true if you want the component to strip off the surrounding paragraph tags that FCKeditor produces
 	 */
 	public FCKEditorPanel(String id,IModel model,String width,String height,String toolbarSet, String collectionId,boolean stripContainingParagraphTags)
 	{
 		this(id,model,width,height,toolbarSet,collectionId);
 		this.stripContainingParagraphTags = stripContainingParagraphTags;
 	}
-	
+
 	/**
 	 * 
 	 * @param id The wicket:id
 	 * @param model The data model
 	 * @param width The width of the rendered textarea
 	 * @param height The height of the rendered text area
-	 * @param toolbarSet The set of toolbars you want rendering, either basic of
-	 * 		             default
+	 * @param toolbarSet The set of toolbars you want rendering, either basic of default
 	 * @param collectionId The Sakai collection id for the FCKeditor to use
 	 */
 	public FCKEditorPanel(String id,IModel model,String width,String height,String toolbarSet, String collectionId)
```

Comment:
This class at least needs a rename. Is it using an old version of FCKEditor or does it use the same CKEditor as the rest of Sakai?

Reply:
It's using Sakai's CKEditor, but in the core Sakai code it's still called FCKEditor (the directory, the launch script, etc.).

Probably fine to leave as is, considering the core code/structure still references it as such.